src/Qoollo.PerformanceCounters/NullCounters/Counters/NullDeltaCounter.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Qoollo.PerformanceCounters.NullCounters.Counters
+{
+    /// <summary>
+    /// Null Counter for the difference between the raw value at the beginning and the end of the measured time frame
+    /// </summary>
+    public sealed class NullDeltaCounter : DeltaCounter
+    {
+        /// <summary>
+        /// Конструктор NullNumberOfItemsCounter
+        /// </summary>
+        /// <param name="name">Имя счётчика</param>
+        /// <param name="description">Описание счётчика</param>
+        public NullDeltaCounter(string name, string description)
+            : base(name, description)
+        {
+        }
+
+        /// <summary>
+        /// Уменьшить значение на 1
+        /// </summary>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long Decrement()
+        {
+            return 0;
+        }
+
+        /// <summary>
+        /// Уменьшить значение на "value"
+        /// </summary>
+        /// <param name="value">Значение, на которое уменьшаем</param>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long DecrementBy(long value)
+        {
+            return 0;
+        }
+
+        /// <summary>
+        /// Увеличить значение на 1
+        /// </summary>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long Increment()
+        {
+            return 0;
+        }
+
+        /// <summary>
+        /// Увеличить значение на "value"
+        /// </summary>
+        /// <param name="value">Значение, на которое увеличиваем</param>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long IncrementBy(long value)
+        {
+            return 0;
+        }
+
+        /// <summary>
+        /// Задать значение
+        /// </summary>
+        /// <param name="value">Новое значение</param>
+        public override void SetValue(long value)
+        {
+        }
+
+        /// <summary>
+        /// Получает текущее значение счетчика производительности
+        /// </summary>
+        public override long CurrentValue
+        {
+            get { return 0; }
+        }
+
+        public override long Measure()

```

Comment:
Please, add the xml-comment to the method (the same as in the base class) to prevent the compilation warning